### PR TITLE
Add children for FixedTabs

### PR DESCRIPTION
**Pull request by tnagorra**

### CODE REVIEW

File: components/View/FixedTabs/index.js [hunk 1]
```diff
@@ -11,13 +11,15 @@ const propTypes = {
...
 
 const defaultProps = {
     active: undefined,
     className: '',
     tabs: [],
     onClick: () => {},
+    children: null,
 };
 
 export default class ScrollTabs extends React.PureComponent {
```
Comment (frozenhelium) [inline components/View/FixedTabs/index.js:25]: Missed the proper class name (ScrollTabs -> FixedTabs)

File: components/View/FixedTabs/index.js [hunk 2]
```diff
@@ -93,7 +95,9 @@ export default class ScrollTabs extends React.PureComponent {
                     data={tabList}
                     modifier={this.renderTab}
                 />
-                <div className={styles.void} />
+                <div className={styles.void}>
```
Comment (frozenhelium) [inline components/View/FixedTabs/index.js:98]: Not a good way to do it, void is there to fill up the empty space, shouldn't be any element in there

File: components/View/FixedTabs/index.js (hunk 1 above)
Comment (tnagorra) [reply components/View/FixedTabs/index.js:25]: I do not understand this statement.

File: components/View/FixedTabs/index.js (hunk 2 above)
Comment (tnagorra) [reply components/View/FixedTabs/index.js:98]: I know. I made it have two purposes. Fix gardeu na, kasari garna parne ho?

**Pull request merged**